sql/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -257,6 +263,11 @@ public String name() {
         if (columnStore == false) {
             mapping.put(DOC_VALUES, "false");
         }
+
+        if (formattedDefaultExpression != null) {
```

Comment:
Maybe add an assertion that `formattedDefaultExpression` can only be null if `defaultExpression` is also null. Otherwise it would probably indicate a programming error as it hasn't been initialized correctly.

Generally I think this `AnalyzedColumnDefinition` could use some work, but that's out of scope of this PR.